app/styles/join-section.module.css

```diff
@@ -0,0 +1,102 @@
+.join {
```

Comment:
Can we please match the font-styling i.e. weight, size, etc as `What is Real Dev Squad` has in all the views either it is mobile, tablet or web